File: chrome/test/data/extensions/api_test/clipboard/test_helper.js
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
document.addEventListener("copy", function() {
  chrome.extension.sendRequest("copy event");
}, false);
document.addEventListener("cut", function() {
  chrome.extension.sendRequest("cut event");
}, false);
document.addEventListener("paste", function() {
  chrome.extension.sendRequest("paste event");
}, false);
chrome.extension.sendRequest("start test");

File: components/media_router/common/test/test_helper.cc
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
// Copyright 2018 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#include "components/media_router/common/test/test_helper.h"

#include "base/callback_helpers.h"
#include "base/memory/ptr_util.h"

namespace media_router {

#if !defined(OS_ANDROID)
TestMediaSinkService::TestMediaSinkService()
    : TestMediaSinkService(base::DoNothing()) {}

TestMediaSinkService::TestMediaSinkService(
    const OnSinksDiscoveredCallback& callback)
    : MediaSinkServiceBase(callback), timer_(new base::MockOneShotTimer()) {
  SetTimerForTest(base::WrapUnique(timer_));
}

TestMediaSinkService::~TestMediaSinkService() = default;
#endif  // !defined(OS_ANDROID)

}  // namespace media_router
